app/views/idv/in_person/verify_wait.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= content_for(:meta_refresh) { @meta_refresh.to_s } %>
+<% title t('titles.doc_auth.processing_images') %>
+
+<%= render PageHeadingComponent.new.with_content(t('doc_auth.info.interstitial_eta')) %>
```

Comment:
I don't really understand what this file is for -- I copied it from the doc auth flow but I'm not sure if we need it. What do you think @aduth?

Reply:
@sheldon-b This was previously needed to support browsers with JavaScript disabled, which would see this page and automatically refresh the page using the `meta_refresh` content. Browsers with JavaScript enabled will only see the "spinner button". We now require JavaScript for identity verification as of LG-6351 (#6405), so it's not really relevant anymore. In fact, I'd planned to refine these steps/views in #6550, but I haven't been able to finish it out since it's a lower-priority clean-up task.

For now, I think you could either keep this as-is to match the existing verify view (although the content is very inaccurate, referencing document capture image processing), or maybe even leave it blank.